mountpoint-s3-crt/src/checksums/crc32.rs

```diff
@@ -0,0 +1,76 @@
+use mountpoint_s3_crt_sys::*;
+
+/// CRC32 checksum
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
Since these are just `u32`s I'd make them Copy too, and then `previous_checksum: Crc32` below